Doc/library/locale.rst

```diff
@@ -374,6 +374,11 @@ The :mod:`locale` module defines the following exception and functions:
    Processes formatting specifiers as in ``format % val``, but takes the current
    locale settings into account.
 
+   .. versionchanged:: 3.7
+   Added *monetary*, if  true the conversion uses monetary thousands separator and
+   grouping strings.
+   Replaces :meth:`format`.
+
 
```

Comment:
The description of the monetary parameter should be added to the regular docs, and the version changed phrase should just say "the monetary keyword parameter was added"